docs/api/drivers/ResetReason.md

```diff
@@ -0,0 +1,39 @@
+## ResetReason
+
+Use the ResetReason interface to access hardware reset reason registers in a portable fashion.
+
+### ResetReason class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/classmbed_1_1_resetreason.html)
+
+### ResetReason example
+
```

Comment:
Can you describe what this example does in a few words?

Reply:
Fixed.